source/lmp/fix_dplr.h

```diff
@@ -54,6 +54,8 @@ class FixDPLR : public Fix {
   double compute_scalar(void) override;
   double compute_vector(int) override;
 
+  enum { NONE, CONSTANT, EQUAL };
```

Comment:
Is there any reason to make it public?

Reply:
I made it public by doing something similar to https://github.com/lammps/lammps/blob/c5d9f901d9e8336602dee9558ffd6a5aab30e4d1/src/fix_efield.h#LL44C20-L44C20. LAMMPS did this because its FixEfield class has a friend class FixQEqReaxFF. In our case, there is no need to make it public. I have changed it to private.